TestCases/rans/oneram6/turb_ONERAM6.cfg

```diff
@@ -25,6 +25,13 @@ FREESTREAM_TEMPERATURE= 288.15
 REYNOLDS_NUMBER= 11.72E6
 REYNOLDS_LENGTH= 0.64607
 
+% --------------------------- VORTICITY_CONFINEMENT ---------------------------%
+%
+% Enable vorticity confinement (YES/NO)
+VORTICITY_CONFINEMENT = NO
+% Set confinement parameter (0.00 by default)
+CONFINEMENT_PARAM = 0.00
```

Comment:
Thank you for the changes, you just need to add a regression test to parallel_regression.py that uses the new feature.